go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/hajimehoshi/go-mp3 v0.3.2
 	github.com/hajimehoshi/oto/v2 v2.0.0-alpha.2
 	github.com/jakecoffman/cp v1.1.0
+	github.com/jezek/xgb v0.0.0-20210312150743-0e0f116e1240 // indirect
```

Comment:
Just out of curiosity, what's the difference between this and the original BurntSushi's xgb?

Reply:
See BurntSushi's readme:

```
Note that this project is largely unmaintained as I don't have the time to do
or support more development. Please consider using this fork instead:
https://github.com/jezek/xgb
```

Both probably would work just fine though.

Reply:
Acknowledged.